docs: Added section about Audit logs (#2828)
mendersoftware/mender-docs: 02.Overview/20.Audit-logs/docs.md

---
title: Audit Logs
taxonomy:
    category: docs
    label: reference
---

> !!!!! Audit logs is only available in the Mender Enterprise plan.

Reviewer: ```suggestion
!!!!! Audit logs are only available in the Mender Enterprise plan.
```

!!!!! See [the Mender plans page](https://mender.io/pricing/plans?target=_blank)
!!!!! for an overview of all Mender plans and features.

> Audit logs improves security by logging important events that can later be analyzed in case of security or operational incidents. The Mender Audit log creates immutable log entries when key events occur, including:

Reviewer: ```suggestion
Audit logs improve security by logging important events that can later be analyzed in case of security or operational incidents. The Mender Audit log creates immutable log entries when key events occur, including:
```

+ Deployment created
+ User created
+ User changed (e.g. role or password)
+ User removed

> In all these cases, a comprehensive set of attributes are logged such as which user performed the action, what the change was, and the time. 

Reviewer: ```suggestion
In all these cases, a comprehensive set of attributes are logged such as which user performed the action, what the change was, and the time it happened. 
```

![Overview](overview.png)

> You can manage the filters by selecting them from the Filter dropdown. 

Reviewer: ```suggestion
You can filter entries in the audit log by clicking the Filter dropdown:
```

![Filters](filter-change.png)

> You can also filter by the user performing the actions.

Reviewer: ```suggestion
You can also filter by the user performing the actions:
```

![Users](filter-user.png)

> When you have chosen the appropriate filters, you can download the results as CSV.

Reviewer: ```suggestion
When you have chosen the appropriate filters, you can download the results as CSV:
```

> ![Download CSV](audit-logs-overview.png)

Reviewer: Might be worth it to mention that audit logs are also available via APIs and link to the appropriate API docs.
